gunicorn.conf.py

```diff
@@ -1,13 +1,25 @@
 """Configuration for the gunicorn WSGI server."""
 import os
-from gunicorn.arbiter import Arbiter
 
-is_dev = os.environ.get("ENV") == "dev"
+from gevent.monkey import patch_all
+from psycogreen.gevent import patch_psycopg
 
+# The "gevent" worker class that we select below uses
+# greenlets under the hood. Greenlets monkeypatch I/O
+# funcionality to support async-io.
+# gevent docs: http://www.gevent.org/
+patch_all()
+patch_psycopg()  # our postgres db driver needs to be patched directly
+
+# Use async workers: https://docs.gunicorn.org/en/stable/design.html#async-workers
+worker_class = "gevent"
 # See https://docs.gunicorn.org/en/stable/settings.html
 port = os.environ.get("PORT", 8080)
-loglevel = "DEBUG" if is_dev else "INFO"
-reload = is_dev
+# Development mode - restart the server on code changes
+reload = os.environ.get("ENV") == "dev"
+# Include debug level-logs if we're in development mode
+loglevel = "DEBUG" if reload else "INFO"
```

Comment:
unrelated, but: not sure why not have DEBUG for all envs.

Reply:
I think this is because gunicorn debug-level logs dump a lot of info that generally hasn't proven helpful. I can't remember an instance where we've encountered a bug on staging or prod whose root cause was masked by excluding these logs.